app/controllers/application_controller.rb

```diff
@@ -22,12 +23,11 @@ def append_to_log(params)
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
     super
-    if @custom_log_items
-      payload[:custom_log_items] = @custom_log_items
-    end
-  end
 
-  # :nocov:
+    append_to_log(request_id: RequestStore.store[:request_id])
+
+    payload[:custom_log_items] = @custom_log_items
```

Comment:
I would find it confusing if I come across a condition that will always evaluate to `true`...